about.html

```diff
@@ -0,0 +1,171 @@
+            </ul>
+        </nav>
+
+        <div class="hearder">
+            <div class="logo-container"><img src="img/mobile_menu 1.png" alt="" class="logo" onclick="showMenu()"></div>
+            <img src="img/logo (1).png" alt="" class="desk-logo">
+            <ul class="logo-menu">
+                <li ><a href="" class="About-color">About</a></li>
+                <li><a href=""></a>Program</li>
+                <li><a href=""></a>Join</li>
+                <li><a href=""></a>Sponsor</li>
+                <li><a href=""></a>News</li>
+                <li class="cc-campaign"><a href="" class="Campaign">CC_Campaign</a></li>
+                
+            </ul>
+            
+        </div>
+    </header>
+    <main>
+        <section class="about-page1">
+            <div class="about-first-page">
+                <div class="about-first-container">
+                    <div class="first-page-title-about">
+                        <h4 class="welcome-title">"Hello! Teaching Staff" </h4>
+                        <h1 class="main-title">COMPUTER SCIENCE PEDAGOGIG SEMINER 2023</h1>
+                    </div>
+                    <div class="page1-text-container">
+                        <p class="page1-text">The Global Summit brings together the community of  
+                            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                            This year, we hope to expand our invitation list including organizations and individuals.</p>
+                    </div>
+                </div>
+                <div class="about-second -container">
+                    <div class="page1-container1">
+                        <p class="page-p1">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+                        <p class="page1-p2">ccsummit2015@cckorea.org</p>
+
+
+                    </div>
+                </div>
+
+            </div>
+
+        </section>
+        <section class="about-page2">
+            <div class="about-page2-container">
+                <div class="about-page2-1">
+                    <div class="about-page2-1-1">
+                        <p class="about-page2-1-1-p">Creative Commons 
+                            Global Summit 2015 Logo</p>
+                        <img src="img/title_underline (2).png" alt="" class="underline">
+                    </div>
+                    <p class="about-page2-2-p">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+                </div>
+           
+                <img src="img/about_logo 1.png" alt="">
+                  
+            </div>
+
+        </section>
+
+        <section class="about-page3">
+            <div class="about-page3-part1">
+                <div class="about-page3-part1-up">
+                    <p class="about-page3-part1-text1">See the past CC Global Summits</p>
```

Comment:
- [ ] 👍 Great job on adding a vertical line after this section's title! However, I noticed that you used a `<p>` tag for this title. To enhance the semantic structure of your HTML, I recommend using a proper heading tag instead of the `<p>` tag.

- [ ] Additionally, in the mobile versiton the spacing before this section title is currenty too small compare to the desin. pelae condier to add more spaces.


By implementing these adjustments, you'll improve the semantic integrity and visual alignment of your project. Keep up the good work! 🌟📝🚀